Gems/ROS2/Code/Source/Manipulation/FollowJointTrajectoryActionServer.cpp

```diff
@@ -81,6 +85,10 @@ namespace ROS2
         if (!IsReadyForExecution())
         {
             AZ_TracePrintf("FollowJointTrajectoryActionServer", "Goal rejected: server is not ready for execution!");
+            if (m_goalHandle)
+            {
+                AZ_TracePrintf("FollowJointTrajectoryActionServer", " m_goalHandle->is_active() = %d m_goalHandle->is_executing() = %d\n", m_goalHandle->is_active(), m_goalHandle->is_executing());
```

Comment:
The AZ_TracePrintf is being deprecated and should replaced with the AZ_Trace
```suggestion
                AZ_Trace("FollowJointTrajectoryActionServer", " m_goalHandle->is_active() = %d m_goalHandle->is_executing() = %d\n", m_goalHandle->is_active(), m_goalHandle->is_executing());
```

Reply:
Thanks for the catch! Will apply in the entire target branch!